documentation/book/assembly-user-operator.adoc

```diff
@@ -9,14 +9,16 @@
 
 :context: deploying-uo
 
-The User Operator provides a way of managing Kafka users via Kubernetes resources.
+The User Operator provides a way of managing Kafka users through custom resources.
```

Comment:
@PaulRMellor "provides a way" sounds a bit vague. I assume it's not the only way to manage users? Maybe we can say something like "The User Operator manages Kafka users through custom resources."?

Reply:
Better. Updated.